README.md

```diff
@@ -232,21 +242,46 @@ my-s3-bucket:
 
 ## Postgres database
 
-A postgres database can be added by configuring the `extensions.yml` file. Below is a simple example configuration, showing some of the available options:
+A postgres database can be added by configuring the `platform-config.yml` file. Below is a simple example configuration, showing some of the available options:
 
 ```yaml
 my-postgres-db:
   type: postgres
   version: 16.2
   environments:
-    "*":
+    '*':
       plan: tiny
       backup_retention_days: 1 # Optional.  Must be between 1 and 35.  If none, defaults to 7.
     prod:
       deletion_protection: true # Optional
       deletion_policy: Retain # Optional: Delete or Retain
 ```
 
+## S3 to S3 data migration module
+
+This module will create cross account permissions to write to your S3 bucket, allowing the copying of files from a source S3 bucket to your destination S3 bucket.  Any cross account data migration **must be approved by cyber**.  Please see the [S3 to S3 data migration documentation](https://platform.readme.trade.gov.uk/reference/terminology/) for further details on using this module.
```

Comment:
Pedantry...
```suggestion
This module will create cross account permissions to write to your S3 bucket, allowing the copying of files from a source S3 bucket to your destination S3 bucket. Any cross account data migration **must be approved by cyber**. Please see the [S3 to S3 data migration documentation](https://platform.readme.trade.gov.uk/reference/terminology/) for further details on using this module.
```

Reply:
I think https://platform.readme.trade.gov.uk/reference/terminology/ might be the wrong URL?

It's cool to put in what it will be, even though it isn't in there just yet.

Reply:
Actual docs link has been added